.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ references:
     test-results: &TEST_RESULTS_DIR /tmp/test-results
 
   cache:
-    yarn: &YARN_CACHE_KEY consul-ui-v5-{{ checksum "ui/yarn.lock" }}
+    yarn: &YARN_CACHE_KEY consul-ui-v6-{{ checksum "ui/yarn.lock" }}
```

Comment:
This is just a cache buster, really doesn't matter what it is as long as its different from the last one. I only ever do this if we get some sort of npm dependency caching problem in CI, and the quickest thing to try is to bump which will sometimes be the solution - other products have this problem sometimes also and its generally what we all do in this case. I half remember this time it was more to do with our workspaces in the end, so theoretically this could have been reverted - but its just a random hash more than anything so no biggie.